rviz_rendering/src/rviz_rendering/mesh_loader_helpers/assimp_loader.cpp

```diff
@@ -246,6 +249,18 @@ std::vector<Ogre::MaterialPtr> AssimpLoader::loadMaterials(
   const std::string & resource_path, const aiScene * scene)
 {
   std::vector<Ogre::MaterialPtr> material_table_out;
+
+  std::string ext = std::filesystem::path(resource_path).extension();
```

Comment:
```suggestion
  std::string ext = std::filesystem::path(resource_path).extension().string();
```

Reply:
Thanks for your suggestion. 
I applied the change .